docs/howto/local-checkouts.md

```diff
@@ -0,0 +1,343 @@
+and identify that there are files to be downloaded from lakeFS.
+```shell
+Successfully cloned lakefs://is-alpaca/experiment-1/dataset/train/ to ~/ml_models/is_alpaca/input
+
+Clone Summary:
+
+Downloaded: 250
+Uploaded: 0
+Removed: 0
+```
+
+Running [`lakectl local list`](../reference/cli.md#lakectl-local-list) from our Git repository root will show that the 
+`input` directory is now in sync with a lakeFS prefix (Remote URI), and what lakeFS version of the data (Synced Commit) 
+the is it tracking:
+```shell
+ is_alpaca % lakectl local list                 
++-----------+------------------------------------------------+------------------------------------------------------------------+
+| DIRECTORY | REMOTE URI                                     | SYNCED COMMIT                                                    |
++-----------+------------------------------------------------+------------------------------------------------------------------+
+| input     | lakefs://is-alpaca/experiment-1/dataset/train/ | 589f87704418c6bac80c5a6fc1b52c245af347b9ad1ea8d06597e4437fae4ca3 |
++-----------+------------------------------------------------+------------------------------------------------------------------+
+```
+
+### Tie Code Version and Data Version 
+{: .no_toc}
+
+Now let's tell Git to stage the dataset we've added and inspect our Git branch status:
+```shell
+is_alpaca % git add input/
+is_alpaca % git status 
+On branch experiment-1
+Changes to be committed:
+  (use "git restore --staged <file>..." to unstage)
+	new file:   input/.lakefs_ref.yaml
+
+Changes not staged for commit:
+  (use "git add <file>..." to update what will be committed)
+  (use "git restore <file>..." to discard changes in working directory)
+	modified:   .gitignore
+```
+
+We can see that the `.gitignore` file changed, and that the files we cloned from lakeFS into the `input` directory are not 
+tracked by git. This is intentional - remember that **lakeFS is the one managing the data**. But wait, what is this special
+`input/.lakefs_ref.yaml` file that Git does track?  
+```shell
+is_alpaca % cat input/.lakefs_ref.yaml
+
+src: lakefs://is-alpaca/experiment-1/dataset/train/s
+at_head: 589f87704418c6bac80c5a6fc1b52c245af347b9ad1ea8d06597e4437fae4ca3
+```
+This file includes the lakeFS version of the **data** that the Git repository is currently pointing to.
+
+Let's commit the changes to Git with:
+```shell
+git commit -m "added is_alpaca dataset" 
+```
+By committing to Git, we tie the current code version of the model to the dataset version in lakeFS as it appears in
+`input/.lakefs_ref.yaml`.
+
+### Experiment and Version Results  
+{: .no_toc}
+
+We ran the train script on the cloned input, and it generated a model. Now, let's use the model to predict whether **SpongeBob** is an Alpaca.
+
+A reminder - this is how SpongeBob looks like - not like an Alpaca!
+
+<img src="../assets/img/lakectl-local/sponge-bob.png" alt="SpongeBob" width="300"/>
```

Comment:
Either find an image that is CC0/public domain licensed (with proper attribution) or use another example :)

Reply:
Also, Axolotls are cuter and are also not alpacas

Reply:
done, replaced with a cute axolotl :)